nixos/tests/installer.nix

```diff
@@ -77,9 +77,9 @@ let
     let iface = if grubVersion == 1 then "ide" else "virtio";
         isEfi = bootLoader == "systemd-boot" || (bootLoader == "grub" && grubUseEfi);
         bios  = if pkgs.stdenv.isAarch64 then "QEMU_EFI.fd" else "OVMF.fd";
-    in if !isEfi && !pkgs.stdenv.hostPlatform.isx86 then
-      throw "Non-EFI boot methods are only supported on i686 / x86_64"
-    else ''
+    in if !isEfi && !pkgs.stdenv.hostPlatform.isx86 then ''
+      machine.succeed("true")
+    '' else ''
```

Comment:
Loos unrelated. Worth submitting separately?

Reply:
This is actually working as intended on the Hydra side, though I wish we had a cleaner separation between _expected_ eval failures and unexpected ones...

Reply:
https://github.com/NixOS/nixpkgs/pull/220308